solana/programs/token-bridge-relayer/src/state/config.rs

```diff
@@ -0,0 +1,44 @@
+use anchor_lang::prelude::*;
+
+/// The program's main account.
+#[account]
+#[derive(InitSpace)]
+pub struct TbrConfigAccount {
```

Comment:
I updated to use `State`. I was using `Account`, because that's the Anchor terminology in the client. It's still called account in the SDK for that very reason.

Also I like to put a suffix, that way it's easy to distinguish the states/accounts with the contexts in one glance.